packages/web-app-files/src/views/spaces/Project.vue

```diff
@@ -0,0 +1,396 @@
+<template>
+  <div class="space-overview">
+    <list-loader v-if="loadSpaceTask.isRunning" />
+    <template v-else>
+      <not-found-message v-if="!space.id" class="space-not-found oc-height-1-1" />
+      <div
+        v-else-if="isSpaceRoot"
+        class="oc-grid oc-grid-match oc-px-m oc-mt-m"
+        :class="{ 'oc-child-width-1-1@s': imageExpanded, 'oc-child-width-1-3@s': !imageExpanded }"
+      >
+        <div v-if="imageContent">
+          <img
+            :class="{ expanded: imageExpanded }"
+            class="space-overview-image oc-cursor-pointer"
+            alt=""
+            :src="'data:image/jpeg;base64,' + imageContent"
+            @click="toggleImageExpanded"
+          />
+        </div>
+        <div>
+          <h3 class="space-overview-name oc-mb-s">{{ space.name }}</h3>
+          <span v-if="space.description">{{ space.description }}</span>
+          <div>
+            <div ref="markdownContainer" class="markdown-container" v-html="markdownContent"></div>
+            <div v-if="showMarkdownCollapse" class="markdown-collapse oc-text-center oc-mt-s">
+              <oc-button appearance="raw" @click="toggleCollapseMarkdown">
+                <oc-icon :name="markdownCollapseIcon" />
+                <span>{{ markdownCollapseText }}</span>
+              </oc-button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <no-content-message v-if="isEmpty" id="files-space-empty" class="files-empty" icon="folder">
+        <template #message>
+          <p class="oc-text-muted">
+            <span v-translate>No resources found</span>
```

Comment:
Could be simplified by setting the `v-translate` and text content to the wrapping `<p>` tag